### DISCUSSION TITLE
How to configure .htaccess so that it works with other PHP software?

### DISCUSSION THREAD

**Discussion by i-do** [datenstrom/community#404] [Ask a question] [answered]
I need to install tinytinyrss rss reader to [domain.com]/tt-rss folder. 
So I need some advice how to make it works without yellow .htaccess rules.

**Comment by schulle4u** [comment, marked as answer]
You can try to add `index.php` as directory index directive to Yellow's .htaccess file. 

```
<IfModule mod_rewrite.c>
RewriteEngine on

DirectoryIndex index.html  index.php yellow.php
RewriteRule ^(cache|content|system)/ error [L]
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^ yellow.php [L]
</IfModule>
```

If this doesn't work, I recommend to install applications like TTRSS under a subdomain with separate virtual host. You also can put Yellow into it's own subdirectory and use a simple php redirect above this directory. 

```
<?php
header("location: ./yellow");
?>
```

HTH
Steffen

**Comment by i-do** [comment]
> try to add `index.php` as directory index

Thanks! This one solve my problem.

**Comment by annaesvensson** [comment]
Feel free to close you own issues when they are solved.
